pages/verify-identity.tsx

```diff
@@ -96,74 +88,107 @@ export default function IdentityVerificationForm() {
+                                src="/assets/wallet-card-example.png"
+                                height={200}
+                                width={320}
+                              />
+                            </PopoverBody>
+                            <PopoverFooter backgroundColor="background.grayHover">
+                              <Text textStyle="caption">
+                                {`Locate your User ID on the Wallet Card that came with your Permit.`}
+                              </Text>
+                            </PopoverFooter>
+                          </PopoverContent>
+                        </Popover>
+                        <FormHelperText>
+                          {`You can find your user ID on the back of your wallet card. If you cannot find your
+                        wallet card, please call RCD at 604-232-2404.`}
+                        </FormHelperText>
+                      </Flex>
+                    </NumberField>
+                  </Box>
+
+                  <Box marginBottom="48px" textAlign={'left'}>
+                    <TextField
+                      name="phoneNumberSuffix"
+                      label="Last 4 digits of your phone number"
+                      required={true}
+                      width="184px"
+                      maxLength={4}
+                    />
+                  </Box>
+
+                  <Box marginBottom="20px" textAlign={'left'}>
+                    <DateField name="dateOfBirth" label="Date of Birth" width="184px">
+                      <FormHelperText>
+                        {`Please enter your date of birth in YYYY-MM-DD format. For example, if you were born on
+                      20th August 1950, you would enter 1950-08-20`}
+                      </FormHelperText>
+                    </DateField>
+                  </Box>
+
+                  {alertError && (
+                    <Box marginY="20px" textAlign={'left'}>
+                      <Alert status="error" marginY="12px">
+                        <AlertIcon />
+                        <AlertDescription>{alertError}</AlertDescription>
+                        {/* TODO: Make alert closable */}
+                        {/* <CloseButton position="absolute" right="8px" top="8px" /> */}
+                      </Alert>
+                    </Box>
+                  )}
+
+                  <Flex width="100%" justifyContent="flex-end">
+                    <Link href="/">
+                      <Button variant="outline" marginRight="12px">{`Go back to home page`}</Button>
+                    </Link>
+                    <Button
+                      type="submit"
+                      variant="solid"
+                      isLoading={loading}
+                      disabled={
+                        loading ||
+                        !values.userId ||
+                        !values.phoneNumberSuffix ||
+                        !values.dateOfBirth
```

Comment:
We should be able to use the `isValid` prop here: https://formik.org/docs/api/formik#isvalid-boolean

Reply:
`isValid` only returns false if there are errors, but there won't be any errors when the form hasn't been filled out yet, so the button will be initially enabled. In this case, clicking it will just show the errors for all fields. Would you prefer this to the current implementation?

Reply:
Ah, that's a good point. Let's keep the current conditions and also disable the button if `isValid` returns false
```suggestion
                        // isValid is false ||
                        !values.userId ||
                        !values.phoneNumberSuffix ||
                        !values.dateOfBirth
```